miden/tests/integration/operations/decorators/advice.rs

```diff
@@ -191,7 +192,7 @@ fn advice_push_mapval() {
 
     let stack_inputs = [1, 2, 3, 4];
     let adv_map = [(
-        key_to_bytes(stack_inputs),
+        RpoDigest::try_from(stack_inputs).unwrap(),
```

Comment:
I believe `key_to_bytes()` function is no long used - so, we can probably remove it.